src/ui/popups/info.js

```diff
@@ -113,6 +113,11 @@ require([
 		$('#album-art').css('background-image', `url("${getCoverArt()}")`);
 	}
 
+	function fillUserPlayCount() {
+		$('#userPlayCountLabel').text(`${i18n('infoYourScrobbles')}:`);
```

Comment:
What about hiding the `.user-play-count` when `getUserPlayCount` returns `0`?

Otherwise could be a bit annoying for people who do not use last.fm.